SEED-platform/seed
# Add filter groups to default reports page

#### What's this PR do?

Adds a selection widget for filter groups to the default reports page.

#### How should this be manually tested?

Create a filter group that narrows your data set.  Run a report with no filter group selected, observe the number of properties, then select a filter group and update the charts - you should see your property count decrease according to the filter group conditions

#### What are the relevant tickets?

#4809 

#### Screenshots (if appropriate)
![image](https://github.com/user-attachments/assets/904be3af-97f9-4f5d-829e-749cb91437ce)
![image](https://github.com/user-attachments/assets/bc25fdc8-d1bb-4194-894a-2eca5a845b5b)
